### ISSUE TITLE
Console execution and queuing for large reports

### ISSUE THREAD

**Issue by iamkeir** [barrelstrength/craft-sprout-reports#125]
Large reports, even when as optimal as possible, can be a pain due to:

1. Craft CP being "locked up" whilst running report
2. HTTP request limits/timeouts

Could this potentially be improved if reports were "requested", but then queued and executed via the console - perhaps with some form of "report completed" notification?

**Comment by BenParizek** [comment]
Good idea. I'll add this as an FR to explore once we get upgraded to Craft 4.
